src/background/init/migrations/__tests__/core-custom-values.test.ts

```diff
@@ -26,14 +26,14 @@ it("sets custom values", () => {
           }
         }
       },
-      Clipboard: {
-        content: "Clipboard content",
+      Notebook: {
```

Comment:
To test that the migration used Notebook as **"clipboard"** (set below) and didn't change it accidentally to Clipboard, the note was renamed here to Notebook to make that clear.